namer/consul/src/main/scala/io/buoyant/namer/consul/SvcAddr.scala

```diff
@@ -35,42 +35,38 @@ private[consul] object SvcAddr {
         retry = true
       ).map(indexedToAddresses)
 
+    lazy val meta = mkMeta(key, datacenter, domain)
```

Comment:
seems pretty cheap and lazy vals make things so much harder to reason about.  probably not a big deal either way in this case but I'd shy away from lazy vals when possible on principle